Document/Document.ipynb

```diff
@@ -12942,7 +13017,9 @@
     "\n",
     "<blockquote id=\"ref74\">74- A. F. Hayes and K. Krippendorff, \"Answering the call for a standard reliability measure for coding data,\" Communication methods and measures, vol. 1, no. 1, pp. 77-89, 2007.</blockquote>\n",
     "\n",
-    "<blockquote id=\"ref75\">75- M. Aickin, \"Maximum likelihood estimation of agreement in the constant predictive probability model, and its relation to Cohen's kappa,\" Biometrics, pp. 293-302, 1990.</blockquote>"
+    "<blockquote id=\"ref75\">75- M. Aickin, \"Maximum likelihood estimation of agreement in the constant predictive probability model, and its relation to Cohen's kappa,\" Biometrics, pp. 293-302, 1990.</blockquote>\n",
+    "\n",
+    "<blockquote id=\"ref76\">76- MacMillan, N.; Creelman, C. (2005). Detection Theory: A User's Guide. Lawrence Erlbaum Associates. ISBN 9781410611147.</blockquote>"
```

Comment:
Hi @sepandhaghighi 
Is this reference compatible in format with others?

Reply:
> Hi @sepandhaghighi
> Is this reference compatible in format with others?

@alirezazolanvari No and I asked you to reformat it in `README.md`, `sensitivity_index` branch

Reply:
Done!

Reply:
@alirezazolanvari
What's that `*` for?